conans/client/toolchain/__init__.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
```

Comment:
We don't want anything in a `__init__.py` file, let's move this into any other file inside this module